preprocess/analysis/mapping.py

```diff
@@ -0,0 +1,49 @@
+"""
+@Author: Scarlet Nguyen 
+"""
+
```

Comment:
A doc comment might be nice to explain where the mapping fits into the SA process